docs/UserGuide.md

```diff
@@ -186,6 +186,19 @@ Format: `view INDEX`
 Examples:
 - `view 1` shows the contact details of the first person shown on the displayed person list.
 
+### Exporting the contact list : `export`
+
+Exports the contacts in the contact list into the specified file type.
```

Comment:
What filename would you be exporting to? Is it hardcoded or can the user input it? Either case, i think it would be good to state it.